csharp-client-sdk/Runtime/Transport.cs

```diff
@@ -0,0 +1,18 @@
+using System.Threading.Tasks;
+
+namespace Hathora
+{
+    public interface Transport
+    {
+
+        public Task Connect(string stateId, string token);
+
+        public Task Write(byte[] data);
+
+        public Task<byte[]> Read();
```

Comment:
`Read()` makes me think this is a lower level API that reads raw bytes off the wire, but in this case I feel like `GetMessage()` or `ReadMessage()` would be more appropriate

Reply:
yes to both